### PR TITLE
[WPB-11368] Test for export team member CSV

### PR DESCRIPTION
**Pull request by battermann**
In preparation for https://github.com/wireapp/wire-server/pull/4293, this moves the CSV export test to the new integration suite.

https://wearezeta.atlassian.net/browse/WPB-11368

## Checklist

 - [x] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [x] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

File: integration/test/Test/Teams.hs [hunk 1]
```diff
@@ -276,3 +280,51 @@ testUpgradePersonalToTeamAlreadyInATeam = do
...
+        now <- formatTime defaultTimeLocale "%Y-%m-%d" <$> liftIO getCurrentTime
+        take 10 (parseField 4) `shouldMatch` now
+        parseField 5 `shouldMatch` (ownerMember %. "handle")
+      parseField 7 `shouldMatch` "wire"
+      parseField 9 `shouldMatch` "foo"
+      parseField 12 `shouldMatch` (mem %. "num_clients")
+  where
+    unquote :: String -> String
+    unquote ('\'' : x) = x
+    unquote x = x
```
Comment (fisx) [inline integration/test/Test/Teams.hs:330]: there could be more `'` in some of those strings.

```suggestion
    unquote :: String -> String
    unquote = init . tail
```

File: integration/test/Test/Teams.hs [hunk 2]
```diff
@@ -276,3 +280,51 @@ testUpgradePersonalToTeamAlreadyInATeam = do
...
+      let parseField = unquote . read . B8.unpack . (cols !!)
+
+      parseField 0 `shouldMatch` (mem %. "name")
+      parseField 1 `shouldMatch` (mem %. "handle")
+      parseField 2 `shouldMatch` (mem %. "email")
+      role <- mem %. "role" & asString
+      parseField 3 `shouldMatch` role
+      when (role /= "owner") $ do
+        now <- formatTime defaultTimeLocale "%Y-%m-%d" <$> liftIO getCurrentTime
+        take 10 (parseField 4) `shouldMatch` now
```
[the remark this answers is not in the document]
Comment (pcapriotti) [reply integration/test/Test/Teams.hs:322]: Yeah, that is a problem, but I hope it's rare enough that it doesn't matter.

File: integration/test/Test/Teams.hs (hunk 1 above)
Comment (pcapriotti) [reply integration/test/Test/Teams.hs:330]: I'm not sure what you're saying here. For whatever reason, we escape these values by adding a quote at the beginning when they start with some "forbidden character". It doesn't make a lot of sense to me, but that's what the exporter was doing before, so I didn't change it. This function simply undoes that escaping.

Comment (fisx) [reply integration/test/Test/Teams.hs:331]: sorry, i was reading it wrong. all good!

**Review by fisx — APPROVED** [review]

**Pull request merged**